linera-service/src/cli_wrappers.rs

```diff
@@ -93,7 +93,7 @@ pub struct ClientWrapper {
 impl ClientWrapper {
     fn new(tmp_dir: Rc<TempDir>, database: Database, network: Network, id: usize) -> Self {
         let storage = match database {
-            Database::RocksDb => format!("rocksdb:client_{}.db", id),
+            Database::RocksDb => format!("rocksdb:{}/client_{}.db", tmp_dir.path().display(), id),
```

Comment:
I don't understand this change. We are about to merge `wallet_{}.json` and `client_{}.db` (which we call `linera.db` in other places btw) but here we seem to separate the two files.

Reply:
Because the exports need to point to an absolute path and there is no downside to having the absolute path representation in memory.